cmd/entrypoint/subcommands/subcommands_test.go

```diff
@@ -39,7 +42,7 @@ func TestProcessSuccessfulSubcommands(t *testing.T) {
 	} {
 		t.Run(tc.command, func(t *testing.T) {
 			returnValue := Process(append([]string{tc.command}, tc.args...))
-			if _, ok := returnValue.(OK); !ok {
+			if !errors.As(returnValue, &ok) {
```

Comment:
I don't think we're using the value of `ok` for any of these - should we just use errors.Is instead?

Reply:
We can't actually do that -- `errors.Is` compares to a _value_, typically a canonical error; `errors.As` compares the error to a _type._ We don't declare any canonical `OK` errors, so we use `errors.As` here to check if it is the `OK` `type`.

Reply:
(of course, I may be misunderstanding your suggestion, in which case please clarify)